### PR TITLE
[W4.4c][T12-2]Cheng Xianhao

### PR DESCRIPTION
**Pull request by xhxh96**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -50,7 +50,11 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |`* * *` |user |add a new person |
 |`* * *` |user |delete a person |remove entries that I no longer need
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |add a picture to a person |better able to recognize who the contact person is
+|`* * *` |user |add labels to person |categorize the contacts
 |`* *` |user |hide <<private-contact-detail, private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * ` |user |add a picture to a contact |better able to recognize who the contact person is
```
Comment (marlenekoh) [inline docs/DeveloperGuide.adoc:56]: slight formatting error here, should be `* *` without the space behind the second asterisk

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -50,7 +50,11 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |`* * *` |user |add a new person |
 |`* * *` |user |delete a person |remove entries that I no longer need
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |add a picture to a person |better able to recognize who the contact person is
```
Comment (marlenekoh) [inline docs/DeveloperGuide.adoc:53]: What is the difference between `adding a picture to a person` and `adding a picture to a contact`?

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -50,7 +50,11 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+|`* * *` |user |add labels to person |categorize the contacts
```
Comment (marlenekoh) [inline docs/DeveloperGuide.adoc:54]: Perhaps this could use a more specific role, such as as a `user who likes to be organized`...

**Review by marlenekoh — COMMENTED** [review]
@xhxh96 Looks good! Please close this PR after reading review comments (:

**Comment by xhxh96** [comment]
Noted with thanks.

**Pull request closed without merging**